servicepulse/usage-config.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Usage Setup
+summary: Viewing endpoint usage summary and generating a usage report
+component: ServicePulse
+reviewed: 2024-05-08
+related:
+  - nservicebus/usage
+---
+
+Specific settings for collecting usage data to generate a usage report.
```

Comment:
Nothing on this page now about why they may not be seeing this page if they are using an older version.
We need the note on here.